src/clj/game/cards/resources.clj

```diff
@@ -43,19 +43,22 @@
 
    "Activist Support"
    {:events
-    {:corp-turn-begins {:req (req (= 0 (:tag runner)))
-                        :msg "take 1 tag"
+    {:corp-turn-begins {:msg "take 1 tag"
```

Comment:
This message is going to trigger every turn, even when runner does not gain a tag. Probably better to move into the effect resolution.